### DISCUSSION TITLE
Unable to "backport" litematicas, but I need to, any help? Willing to do anything (even mess with coding)

### DISCUSSION THREAD

**Discussion by Ianoneshot** [FabricMC/fabric-api#2333] [Player support]
Good morning, afternoon, or night,

Me and my friend have been making a souls-like map for about a year now. I made working bosses in a 1.16.5 version of the map (as well as stuff like working "bonfire" mechanics and rolling system with animations). However, he is the builder, and accidentally updated the world to a 1.19 version over the years. In other words, he has been placidly updating it not realizing backports of full worlds are not possible.
So I devised a plan, "I'll use 1.18 litematica to make schematics of the already-existing content he made, and paste them into the 1.16.5 version!" After all, he did not use any block from anything that stemmed from 1.17-1.19 .
However, I tried this tactic, and it says "unsupported future version (6)" when I try to load the litematica schematics into the 1.16.5 version of the world.
We worked so hard on this adventure map, it even has custom music and resources! We _do not_ want it to go all to waste now because of a mistake. Is there a way to bypass the "unsupported future version" of the schematic, even if the schematic might not technically be "complete"? We are willing to _take anything at this point_, and _we are really desperate for some solution._

Thank you so much for reading, any help will be greatly appreciated.

Sincerely,
Ianoneshot and UndefinedTactics

**Comment by warjort** [comment]
The suggested answer has already posted on your litematics issue
https://github.com/maruohon/litematica/issues/543

i.e. Use an nbt editor to change the version number in a copy of your file.
Of course, it might still fail to load in 1.16.5 if you have used blocks from 1.17+?
Then you will need to edit those blocks to substitute a different block identifier.
